protos/api.v25.proto

```diff
@@ -96,3 +98,20 @@ message Namespace {
   string name = 1;
   uint64 id = 2;
 }
+
+message InitiateSnapshotStreamRequest {}
+
+message InitiateSnapshotStreamResponse {
```

Comment:
Now we know that we can't directly talk to leader alpha, we need to modify the protocol to work with any alpha.

Reply:
So now I think we should just return task_id and groups, but the problem is how we are going to connect with another group's alpha to stream the p directory for that group.